src/logs_bot.py

```
@@ -143,31 +151,39 @@ def logs_by_day(request):
+        data_logs.setdefault(day, {"day":day, "title_count":0 , "results": {"no_result": 0, "Category": 0}})
+        # ---
+        data_logs[day]["title_count"] += x["title_count"]
         # ---
-        data_logs[day][x["status_group"]] = x["count"]
+        data_logs[day]["results"][x["status_group"]] = x["count"]
     # ---
     logs = []
     # ---
     sum_all = 0
     # ---
     for day, results_keys in data_logs.items():
```

Ruff (0.11.9)

Loop control variable `day` not used within loop body

(B007)